internal/appsvc/jmap.go

```
@@ -0,0 +1,136 @@
+	NotDestroyed map[string]string                `json:"not_destroyed,omitempty"`
+}
+
+type JMAPBlob struct {
+	ID   string `json:"id"`
+	Type string `json:"type"`
+	Size int64  `json:"size"`
+}
+
+func (services *Services) JMAPQuotas(ctx context.Context) ([]JMAPQuota, error) {
+	reader, ok := services.query.(interface {
+		ObjectBreakdown(context.Context) (contracts.ObjectBreakdown, error)
```

golangci-lint (2.12.2)

[error] interface method ObjectBreakdown must have named param for type context.Context

(inamedparam)